### ISSUE TITLE
Unable to connect after update to 1.1.3

### ISSUE THREAD

**Issue by boekkooi** [yakyak/yakyak#130]
Hi there,

So I just update to the latest version and now I'm unable to get a connection or something.
Here is this log:
```
input.coffee:43 Why is this running multiple times?
input.coffee:46 Creating selector for people
input.coffee:46 Creating selector for nature
input.coffee:46 Creating selector for food
input.coffee:46 Creating selector for activity
input.coffee:46 Creating selector for travel-places
input.coffee:46 Creating selector for objects
input.coffee:46 Creating selector for symbols
input.coffee:46 Creating selector for flags
/opt/yakyak/resources/atom.asar/renderer/api/lib/remote.js:109 Uncaught Error: Cannot read property 'position' of undefined
TypeError: Cannot read property 'position' of undefined
    at Function.Menu.buildFromTemplate (/opt/yakyak/resources/atom.asar/browser/api/lib/menu.js:291:15)
    at callFunction (/opt/yakyak/resources/atom.asar/browser/lib/rpc-server.js:148:18)
    at EventEmitter.<anonymous> (/opt/yakyak/resources/atom.asar/browser/lib/rpc-server.js:234:14)
    at emitMany (events.js:108:13)
    at EventEmitter.emit (events.js:182:7)
    at EventEmitter.<anonymous> (/opt/yakyak/resources/atom.asar/browser/api/lib/web-contents.js:97:23)
    at emitTwo (events.js:87:13)
    at EventEmitter.emit (events.js:172:7)
input.coffee:43 Why is this running multiple times?
```

I just keeps looping with the `input.coffee:43 Why is this running multiple times?` as start.

I'm running ubuntu 15.10 with GNOME Shell 3.16.4 and Node 4.3.0

I hope you can help me fix this issue.

Cheers,
Warnar

**Comment by Georgeto** [comment]
The first login worked fine for me, but since i closed YakYak and started it again, i get the same error.

**Comment by Georgeto** [comment]
I found the [responsible piece of code](https://github.com/yakyak/yakyak/blob/8b52b07ff2d0e7f657598d3aa2eeeb170602eb91/src/ui/views/trayicon.coffee#L42-L43), which running on windows produces an invalid menu template.

![buggymenu](https://cloud.githubusercontent.com/assets/9250103/12975706/1f8d1f04-d0be-11e5-99ec-3d7d307b06f0.PNG)

**Comment by Georgeto** [comment]
Should be fixed by a0aa0cee05727cbe7204964f983e1074fe378ce6
